infra/templates/backend-state-setup/terraform.tfvars

```diff
@@ -0,0 +1,2 @@
+name="cobaltfstate"
+location="<az-resource-group-location>"
```

Comment:
How does this value get inflated? Seems easy to miss and accidentally get this default string as our location.